src/index.css

```diff
@@ -3,9 +3,10 @@
   line-height: 1.5;
   font-weight: 400;
 
-  color-scheme: light dark;
+  /*color-scheme: light dark;
```

Comment:
the thougt process is right, you should implement a mini-design system for your variables (not in this file though)